script/build-artifact.sh

```diff
@@ -77,13 +77,13 @@ if [ $OS == "windows" ]; then
     (cd "${TMPDIR}/spire"; zip -rv "${ARTIFACT}" -- *)
     (cd "${TMPDIR}/spire-extras"; zip -rv "${EXTRAS_ARTIFACT}" -- *)
 
-    echo "$(CertUtil -hashfile "${ARTIFACT}" SHA256 | sed -n '2p') $(basename "${ARTIFACT}")" > "${CHECKSUM}"
-    echo "$(CertUtil -hashfile "${EXTRAS_ARTIFACT}" SHA256 | sed -n '2p') $(basename "${EXTRAS_ARTIFACT}")" > "${EXTRAS_CHECKSUM}"
+    (cd "$(dirname "${ARTIFACT}")"; CertUtil -hashfile "$(basename "${ARTIFACT}")" SHA256 > "${CHECKSUM}")
```

Comment:
To keep potential error output tidy and accurate, in each of these new subshell invocations, we will want to use `&&` rather than `;` to separate the commands, such that the command to generate the hash file is not invoked if we cannot even change to the directory where the artifact is located (maybe because of a bug where `ARTIFACT` is not set properly).
```suggestion
    (cd "$(dirname "${ARTIFACT}")" && CertUtil -hashfile "$(basename "${ARTIFACT}")" SHA256 > "${CHECKSUM}")
```

Reply:
I take it back, I see that we are setting `set -e` in this script, so this is good as written.